sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -685,10 +685,10 @@ public virtual async Task<Response<AnalyzeSentimentResult>> AnalyzeSentimentAsyn
                     case 200:
                         IDictionary<string, int> map = CreateIdToIndexMap(inputs);
                         AnalyzeSentimentResultCollection results = await CreateAnalyzeSentimentResponseAsync(response, map, cancellationToken).ConfigureAwait(false);
-                        if (results[0].ErrorMessage != default)
+                        if (results[0].Error.Code != default)
```

Comment:
`TextAnalyticsError` is a class. Can it be `null` here?

Reply:
Changing this to a struct.